plan_delegator/src/plan_delegator.cpp

```diff
@@ -460,9 +447,10 @@ namespace plan_delegator
             
             auto plan_response = client->async_send_request(plan_req);
             
+            auto future_status = plan_response.wait_for(std::chrono::milliseconds(200));
```

Comment:
Let's do 100ms. If it can't return in that time, then it really should move onto the next planning cycle.

Reply:
Made it 100